tests/Integration/Api/ApiV3Test.php

```diff
@@ -1347,6 +1352,85 @@ public function testNewSubmission() {
 		], $data['submissions'][0]);
 	}
 
+	/**
+	 * @dataProvider dataNewSubmission
+	 */
+	public function testUpdateSubmission() {
+
+		$uploadedFileResponse = $this->http->request('PUT',
```

Comment:
This files need to be POSTed, there's not PUT route available for updating/changing them :)

```suggestion
		$uploadedFileResponse = $this->http->request('POST',
```

Reply:
As an alternative you could implement a function to update/change the files on updating the submission. Then the old files should be replaced by the new ones

Reply:
it seems to work now